scripts/windows/common.ps1

```diff
@@ -0,0 +1,28 @@
+Set-PSDebug -Trace 1
+$MY_SCRIPT_DIR = Split-Path $script:MyInvocation.MyCommand.Path
+
+function extra_cflags_build_arg() {
+    # If this isn't true then let build use default
+    if ( $DEBUG -eq 'true' ) {
+      return "--build-arg extra_cflags='-g -O0'"
+    }
+}
+
+function do_docker_build([string]$image_name, 
+                         [string]$image_tag,
+                         [string]$target_stage,
+                         [string]$agent_version,
+                         [string]$operating_system) {
+    $agent_version = if ($agent_version -eq "") { $image_tag }
+    $operating_system = if ($operating_system -eq "") { "windows" }
+    echo "$MY_SCRIPT_DIR"
+    $cache_flags = if ($PULL_CACHE -eq "yes") { "$MY_SCRIPT_DIR\..\docker-cache-from $target_stage" } else { "" } 
```

Comment:
Can it run shell scripts like this?  If we have a dedicated windows build machine, we probably won't need the docker cache on quay.io since the machine will maintain a persistent cache unlike CircleCI.

Reply:
yeah i missed that this was invoking other shell scripts.  I'm going to rip out the cache flags for now and if I get a moment, i might port the scripts that this depends on and add it back.